test/e2e/framework/pod/wait.go

```diff
@@ -504,6 +504,31 @@ func WaitForPodNameUnschedulableInNamespace(ctx context.Context, c clientset.Int
 	})
 }
 
+// WaitForPodNameVolumeCondition returns an error if it takes too long for the pod to become Pending
+// and have condition Status equal to Unschedulable,
+// if the pod Get api returns an error (IsNotFound or other), or if the pod failed with an unexpected reason.
+// Typically called to test that the passed-in pod is Pending and Unschedulable.
+func WaitForPodNameVolumeCondition(ctx context.Context, c clientset.Interface, podName, namespace string) error {
```

Comment:
This function is not used?

Reply:
These two functions were previously used. I will remove them now.